src/hooks.spec.tsx

```diff
@@ -176,52 +193,47 @@ describe('hooks', () => {
     it('should return a variation when activate returns a variation', async () => {
       activateMock.mockReturnValue('12345');
 
-      const { container } = render(
+      render(
         <OptimizelyProvider optimizely={optimizelyMock}>
           <MyExperimentComponent />
         </OptimizelyProvider>
       );
-      await optimizelyMock.onReady();
       await waitFor(() => expect(screen.getByTestId('result')).toHaveTextContent('12345|true|false'));
     });
 
     it('should return null when activate returns null', async () => {
       activateMock.mockReturnValue(null);
       featureVariables = {};
-      const { container } = render(
+      render(
         <OptimizelyProvider optimizely={optimizelyMock}>
           <MyExperimentComponent />
         </OptimizelyProvider>
       );
-      await optimizelyMock.onReady();
       await waitFor(() => expect(screen.getByTestId('result')).toHaveTextContent('null|true|false'));
     });
 
     it('should respect the timeout option passed', async () => {
       activateMock.mockReturnValue(null);
+      mockDelay = 100;
       readySuccess = false;
 
       render(
         <OptimizelyProvider optimizely={optimizelyMock}>
-          <MyExperimentComponent options={{ timeout: mockDelay }} />
+          <MyExperimentComponent options={{ timeout: mockDelay - 10 }} />
```

Comment:
NIT: What do you think if we set the `mockDelay` to 0?  Is it effectively the same? 🤔

Reply:
The value of `mockDelay` matters here. The "Client Mock" will be ready after this amount of time (miliseconds). Setting timeout less than that will trigger timeout. There is no specific logic to pick 100 for `mockDelay`. Its just a good number 😄

Reply:
I have decreased the value of milliseconds from 100 to 20 just to make the test suite a little faster